.github/workflows/pytest-petsc.yml

```diff
@@ -9,10 +9,10 @@ on:
   # but only for the master branch
   push:
     branches:
-      - master
+      - separate-eqn
```

Comment:
I don't think you need to do this, because CI gets triggered as soon as you open a PR, given the three lines below this